docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -67,7 +67,7 @@ The `spec` section contains the requirements and attributes and has the followin
 | description                                                          | string                                                                                                     | A textual description of the function                                                                                                                                                                                                                                                                             |
 | handler                                                              | string                                                                                                     | The entry point to the function, in the form of `package:entrypoint`; varies slightly between runtimes, see the appropriate runtime documentation for specifics                                                                                                                                                   |
 | runtime                                                              | string                                                                                                     | The name of the language runtime - `golang` \ `python:3.7` \ `python:3.8` \ `python:3.9` \ `python:3.10` \ `python:3.11` \ `shell` \ `java` \ `nodejs`                                                                                                                                                            | 
-| <a id="spec.image"></a>image                                         | string                                                                                                     | The name of the function's container image &mdash; used for the `image` [code-entry type](#spec.build.codeEntryType); see [Code-Entry Types](/docs/reference/function-configuration/code-entry-types.md#code-entry-type-image)                                                                                    |
+| <a id="spec.image"></a>image                                         | string                                                                                                     | The name of the function's container image &mdash; used for the `image` [code-entry type](#spec.build.codeEntryType); see [Code-Entry Types](code-entry-types.html#code-entry-type-image)                                                                                                                         |
```

Comment:
no need for relative path?

Reply:
@TomerShor this file is in the same directory, so it is already a relative path